guides/doc-Content_Management_Guide/topics/Managing_Activation_Keys.adoc

```diff
@@ -253,8 +253,26 @@ Configure Red Hat Subscription Manager to use {ProjectServer} for registration a
 
 .Procedure
 
-To use an activation key for host registration with an existing Red Hat Enterprise Linux 7 host to {ProjectServer}, complete the following steps:
+To use an activation key for host registration with an existing Red Hat Enterprise Linux 7 host to {ProjectServer}, use one of the following procedures:
 
+==== Global Registration
+. Generate registration command
++
+UI: Go to Hosts > Register Host, select activation key(s) and generate registration command.
```

Comment:
```suggestion
UI: In the {ProjectWebUI}, navigate to *Hosts > Register Host*, select one or more Activation Keys and generate the registration command.
```

Please replace the "UI" part and do not add "(s)" to pluralize a word: see IBM style guide "parentheses to form plurals".